app/policies/user_policy.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# Policies for UsersController
+class UserPolicy < ApplicationPolicy
+  def show?
+    true
+  end
+
+  def show_current_user?
+    !!current_user
+  end
+
+  def email?
```

Comment:
One more addition: `access_private_data?` because all of these methods are verbs/actions.